### DISCUSSION TITLE
Hook up to Dexie.js

### DISCUSSION THREAD

**Discussion by HEAVYPOLY** [vanjs-org/van#369] [Q&A] [answered]
https://dexie.org/docs/liveQuery()

Hi, I'm wondering which way you would recommend using dexie js (indexeddb wrapper) with vanJS?

would liveQuery observable promise go into vanX.reactive(liveQuery (
  () => db.friends
    .where('age')
    .between(50, 75)
    .toArray()
);
)
?

**Comment by Tao-VanJS** [comment, marked as answer]
Just use `subscribe` as they suggested?

```js
const friends = vanX.reactive([])
const friendsObservable = liveQuery (
  () => db.friends
    .where('age')
    .between(50, 75)
    .toArray()
)

// Subscribe
const subscription = friendsObservable.subscribe({
  next: result => vanX.replace(friends, result),
  error: error => console.error(error)
})
```

**Comment by HEAVYPOLY** [comment]
Ok, thank you!
